app/models/application_merits_task/parties_mental_capacity.rb

```diff
@@ -0,0 +1,5 @@
+module ApplicationMeritsTask
+  class PartiesMentalCapacity < ApplicationRecord
+    belongs_to :legal_aid_application
+  end
+end
```

Comment:
I'm slightly confused by the modelling here, but I'm probably missing context.

What does a `PartiesMentalCapacity` represent? 

Can an application have many different parties? I'm assuming it can, and that an `Opponent` is a particular type of party. But what are the different types of parties?

Reply:
A court case has _many_ parties, at base level and applicant and respondent - in civil law.
This question used to be phrased, `Does the Opponent have the mental capacity to understand the case?`  
It was changed a few months ago, when we implemented Client Involvement Type to, `Do all parties have the mental capacity to understand the case?` to avoid having to have lots of branching locales for `Does your client, the Applicant...`, `Does the respondent...`, etc

This is why I renamed it to PartiesMentalCapacity, rather than OpponentsMentalCapacity

Reply:
Ok, that makes sense. Thanks for explaining 👍